cloth-diapers/wash-routine.md

```diff
@@ -0,0 +1,139 @@
+# Wash routine
+
+> The wash routine for cloth diapers is the **main source** of _every_ cloth diaper issue from smells to rashes to leaks.
+
+The cloth diaper groups online are ***filled*** with people bemoaning issues with their cloth diapers that turn into nothing but 'let's trouble shoot your wash routine' posts.
+
+***The problem is simple: there is a HUGE amount of misinformation out
+there on what a wash routine should be***.
+
+And yet, the basis of a good wash routine is _extremely_ simple.
+
+## A Cloth Diaper Wash Routine - Start Here
+
+Wash your diapers at the end of every 2-3 days.
+
+### 1. Cold rinse, no detergent ('prewash')
+Two important things to know here:
++ The key reason we're doing this step is to **rinse as much of the
+human waste** out of those diapers as possible before actually washing them
+  + We ***don't use detergent*** because we're not trying to clean them yet;
+  the detergent is in the main wash and dosed appropriately for a
+  heavily soiled load. If we start using detergent in prewashes we
+  run the risk of ***detergent build-up*** (another common cloth diaper
+    horror story)
++ I do this as a **completely separate wash** rather than just hitting the
+'prewash'button on my machine. This is for two reasons:
+  1. Many modern machines 'efficiently' don't drain the _full_ amount  of the
+  prewash water out of the machine before starting the main wash. this is the
+  _opposite_ of what we need - we want **all of that :poop: water drained**
+  _before_ out main wash starts with clean water
+  2. Using a separate wash often uses a bit more water and is a little longer
+  than a 'prewash' which is what we want.
+
+### 2. Hot main wash, full detergent, extra rinses
+This is your main wash. It is a LONG cycle (often more than 3, even 4 hours)
+but remember, you're cleaning full on human waster out of cloth.
+
+**You want to have as much water as possible** in this wash so look up your
+machine's instructions and figure out which program this is for you - often
+it is the cotton program.
+Yes, I understand this isn't as eco-friendly as the alternative but trust me
+when I say a few more litres of water twice a week is much better than giving
```

Comment:
it would be _far_ better if you actually _quantified_ all the numbers to make it clearer for everyone.
i.e. calculate the _total_ environmental cost of cloth diapers and compare them to the plastic land-fill ones.

Reply:
I am not in a position to do all these calculations unless we're writing a public-facing blog and we're getting some return from it.

It's a complex set of calculations that include MANY factors and hard to find manufacturing information. DEFRA has currently been working on updating their 2008 report for over 6 months 🤷‍♀️ 

This would be super cool but sadly not a high priority right _now_.